tests.py

```diff
@@ -78,7 +81,8 @@ def test_get_sld_from_list_with_exception_rule(self):
 
     def test_get_sld_from_list_with_fqdn(self):
         psl = publicsuffix.PublicSuffixList(['com'])
-        assert 'example.com' == psl.get_sld('example.com.')
+        # 'example.com.' -> <example>.<com>.<empty> -> None, empty labels are not allowed
+        assert None == psl.get_sld('example.com.')
```

Comment:
didn't follow this comment. is this failing because of the trailing dot? I read the code as the get_sld() version will remove the trailing dot ... so why doesn't this go to .. 
example.com -> example, com -> example.com 
??